src/abstraction/transient.rs

```diff
@@ -700,7 +700,7 @@ impl TransientKeyContextBuilder {
         context.set_sessions((Some(session), None, None));
 
         let root_key_handle = context
-            .create_primary_key(
+            .create_primary(
                 self.hierarchy,
```

Comment:
Wonderful to see the names being what you expect them to be!

Reply:
Agreed, this has always bugged me!